modules/distribution-zones/src/test/java/org/apache/ignite/internal/distributionzones/DistributionZoneManagerScaleUpTest.java

```diff
@@ -672,7 +689,7 @@ void testUpdateZoneScaleUpTriggersDataNodePropagation() throws Exception {
 
         assertDataNodesForZone(1, Set.of(), keyValueStorage);
 
-        assertZoneScaleDownChangeTriggerKey(1, 1, keyValueStorage);
+        assertZoneScaleDownChangeTriggerKey(3L, 1, keyValueStorage);
```

Comment:
This check is useless. Let's check that this key become non-null after zone creation or remove check completely

Reply:
please check also the same logic below

Reply:
Fixed. I decided to remove this check.